modules/js/test/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "bin",
```

Comment:
I don't think it should be the current directory name. But name cannot use "." and "space". Let's use "opencv_js_tests". Please try that.